commands/verify/verify.go

```diff
@@ -41,10 +44,62 @@ var Verify = &cobra.Command{
 	Use:   "verify",
 	Short: "verify if the actual data match the expected data",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if actual != "" && expected != "" {
-			return verifier.VerifyDataFile(actual, expected)
+		if expected != "" {
+			return verifySingleCase(expected, actual, query)
 		}
-		fmt.Println("Not implemented.")
-		return nil
+		// If there is no given flags.
+		return verifyAccordingConfig()
 	},
 }
+
+func verifySingleCase(expectedFile, actualFile, query string) error {
+	expectedData, err := util.ReadFileContent(expectedFile)
+	if err != nil {
+		return fmt.Errorf("failed to read the expected data file: %v", err)
+	}
+
+	var actualData, sourceName string
+	if actualFile != "" {
+		sourceName = actualFile
+		actualData, err = util.ReadFileContent(actualFile)
+		if err != nil {
+			return fmt.Errorf("failed to read the actual data file: %v", err)
+		}
+	} else if query != "" {
+		sourceName = query
+		actualData, err = util.ExecuteCommand(query)
+		if err != nil {
+			return fmt.Errorf("failed to execute the query: %v", err)
+		}
+	}
+
+	if err = verifier.Verify(actualData, expectedData); err != nil {
+		logger.Log.Warnf("failed to verify the output: %s\n", sourceName)
+		if me, ok := err.(*verifier.MismatchError); ok {
+			fmt.Println(me.Error())
```

Comment:
Can we use `logger.Log.Errorf` to print the error?

Reply:
It will print the diff actually.
https://github.com/apache/skywalking-infra-e2e/blob/10a26ec9858adfe8469e9d2ebdbd697b86c3d50d/internal/components/verifier/verifier.go#L39-L44